src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -211,7 +203,9 @@ object Directory {
   }
 
   @tailrec
-  private def createEmptyDirectory(path: Path, subDirs: Seq[Directory] = Seq()): Directory = {
+  private[hyperspace] def createEmptyDirectory(
```

Comment:
Can this be reverted now?

Reply:
It's still required since this is Directory object